docs/UserGuide.md

```diff
@@ -218,37 +224,35 @@ Scroll through your history of commands using the `↑` or `↓` arrow keys.
 
 <br>
 
-### Edit a person : `edit INDEX ...` <a name="edit"></a>
+### Edit a contact : `edit INDEX ...` <a name="edit"></a>
 
 Edits an existing contact on SociaLite.
 
 <div markdown="block" class="alert alert-primary">
 **:mag_right: Format:**
 `edit INDEX [n/NAME] [p/PHONE] [r/REMARK] [t/TAG]…​ [date/NAME:YYYY-MM-DD[:monthly|:yearly]]…​ [fb/FACEBOOK] [ig/INSTAGRAM] [tele/TELEGRAM] [tiktok/TIKTOK] [twitter/TWITTER]`
-* Edits the person at the specified `INDEX`. The index refers to the index number shown on the displayed person list. 
-* The index **must be a positive integer** 1, 2, 3, …​
-* At least one of the optional fields must be provided.
-* Users can opt to change any fields associated with a contact as long as they prepend the argument with the corresponding flags according to the format above.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the person will be removed i.e adding of tags is not cumulative.
-* You can remove all the person’s tags by typing `t/` without
-    specifying any tag after it.
-* When editing dates, all existing dates of the person will be removed i.e adding of dates is not cumulative.
-* You can remove all the person’s dates by typing `date/` without
-  specifying any date after it.
+* Edits the contact at the specified `INDEX`. The index refers to the index number shown on the displayed contact list. 
+* The index must be an **integer ranging from 1 to the last index of the contact list** 
```

Comment:
I think this looks good! But might have to standardise with Zac's part for commands as well e.g. pin command, picture command

Reply:
Yep I think we have to standardise the phrasing for repeated lines such as "The index must be a positive integer" and the like, as all 3 of us might have had different ways of phrasing it.